sphinx/ext/autosummary/templates/autosummary/base.rst

```diff
@@ -1,5 +1,4 @@
-{{ fullname }}
-{{ underline }}
```

Comment:
The takeaway here is that all user code that overrides these templates will need to make this change too in order to actually fix the bug. But that's unavoidable.

Either way, this PR will not change how old code behaves.